build/yaml/functionalTestsBotsBuild.yml

```diff
@@ -0,0 +1,233 @@
+      name: 'Microsoft.BotFramework.Composer.Functions.csproj'
+    packages:
+      Microsoft.Bot.Builder
+      Microsoft.Bot.Builder.AI.Luis
+      Microsoft.Bot.Builder.AI.QnA
+      Microsoft.Bot.Builder.ApplicationInsights
+      Microsoft.Bot.Builder.Azure
+      Microsoft.Bot.Builder.Dialogs.Adaptive
+      Microsoft.Bot.Builder.Dialogs.Debugging
+      Microsoft.Bot.Builder.Dialogs.Declarative
+      Microsoft.Bot.Builder.Integration.ApplicationInsights.Core
+      Microsoft.Bot.Builder.Integration.AspNet.Core
+      Microsoft.Bot.Builder.Dialogs
+      Microsoft.Bot.Connector
+
+- template: deployBotResources/dotnet/installDependencies.yml
+  parameters:
+    source: "$(DependenciesSource)"
+    version: "$(DependenciesVersionNumber)"
+    project:
+      directory: 'Bots/DotNet/Skills/Composer/EchoSkillBotComposer/runtime/customaction/'
+      name: 'Microsoft.BotFramework.Composer.CustomAction.csproj'
+    packages:
+      Microsoft.Bot.Builder.Dialogs.Adaptive
+
+- template: deployBotResources/dotnet/installDependencies.yml
+  parameters:
+    source: "$(DependenciesSource)"
+    version: "$(DependenciesVersionNumber)"
+    project: 
+      directory: 'Bots/DotNet/Skills/Composer/EchoSkillBotComposer/runtime/core/'
+      name: 'Microsoft.BotFramework.Composer.Core.csproj'
+    packages:
+      Microsoft.Bot.Builder
+      Microsoft.Bot.Builder.AI.Luis
+      Microsoft.Bot.Builder.AI.QnA
+      Microsoft.Bot.Builder.ApplicationInsights
+      Microsoft.Bot.Builder.Azure
+      Microsoft.Bot.Builder.Azure.Blobs
+      Microsoft.Bot.Builder.Dialogs.Adaptive
+      Microsoft.Bot.Builder.Dialogs.Debugging
+      Microsoft.Bot.Builder.Dialogs.Declarative
+      Microsoft.Bot.Builder.Integration.ApplicationInsights.Core
+      Microsoft.Bot.Builder.Integration.AspNet.Core
+      Microsoft.Bot.Builder.Dialogs
+      Microsoft.Bot.Connector
+
+- task: DotNetCoreCLI@2
+  displayName: 'Build composer bots'
+  inputs:
+    command: publish
+    publishWebProjects: false
+    projects: |
+      Bots/DotNet/**/Composer/**/runtime/azurewebapp/Microsoft.BotFramework.Composer.WebApp.csproj
+    modifyOutputPath: false
+    zipAfterPublish: false
+# End of Restore & Build for Composer
+
+# Start of Restore & Build for V3
+- task: NuGetCommand@2
+  displayName: 'Restore V3 bot'
+  inputs:
+    restoreSolution: 'Bots/DotNet/Skills/CodeFirst/EchoSkillBot-v3/EchoSkillBot-v3.csproj'
+    restoreDirectory: '$(SolutionDir)packages'
+
+- task: MSBuild@1
+  displayName: 'Build V3 bot'
+  inputs:
+    solution: 'Bots/DotNet/Skills/CodeFirst/EchoSkillBot-v3/EchoSkillBot-v3.csproj'
+    vsVersion: 16.0
+    platform: 'AnyCPU'
+    configuration: '$(BuildConfiguration)'
+# End of Restore & Build for V3
+
+- task: PublishBuildArtifacts@1
+  displayName: 'Publish Artifact: build folder'
+  inputs:
+    PathtoPublish: build
+    ArtifactName: build
+
+- powershell: |
+   cd ..
+   ls -R
+  displayName: 'Dir workspace'
+  continueOnError: true
```

Comment:
Question: Should we continue on error?

Reply:
yes we can make it to false to stop on errors